### DISCUSSION TITLE
Translate to PT-BR

### DISCUSSION THREAD

**Discussion by fabiobarbosa1984** [BernhardSchlegel/BierBot-Bricks#15] [Ideas & feature wishlist]
Hi! I would like to volunteer to translate the platform to Brazilian Portuguese. 

Just send me the path and I will be happy to contribute!

**Comment by BernhardSchlegel** [comment]
Hey Fabio, 

thanks for the super-generous offer! ❤️  I'll reach out to you on a private channel ! I can offer to send you a Brick for free as "Thank you".

Cheers,
Bernhard.
